.circleci/config.yml

```diff
@@ -9,14 +9,14 @@ references:
     persist_to_workspace: true
     # https://aquasecurity.github.io/trivy/v0.55/getting-started/installation/#install-script
     cmd: |
-      curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b . v0.55.0
+      curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b . v0.55.1
```

Comment:
Unfortunately I found no way of using a YAML anchor to have a single place in this file where the version is defined.